src/Symfony/Component/Notifier/Notification/ChatNotificationInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Notification;
+
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Receiver\Receiver;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 5.0
+ */
+interface ChatNotificationInterface
```

Comment:
It seems this interface is never implemented
and it's the same for `SmsNotificationInterface`, and `EmailNotificationInterface`

Reply:
These interfaces are not used internally but must be used by developers to override the default rendering of messages.